### PR TITLE
Update installation instructions for f30 and rawhide

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
partial fix of https://github.com/cri-o/cri-o/issues/2713

File: index.md [hunk 1]
```diff
@@ -17,7 +17,11 @@ Join #cri-o on IRC (freenode)
 
 ### Distribution Packaging
 
-* **Fedora**: Available on all supported Fedora versions 28, 29 and Rawhide. 
+* **Fedora**: Available on all supported Fedora versions.
+	* Fedora 30 and Rawhide
```
Comment (TomSweeneyRedHat) [inline index.md]: do we need a "And later" type of thing here.  Otherwise when F31 comes out in a few weeks you'll have to remember to up date this.

Comment (haircommander) [reply index.md]: ooh lower maintenance burden? I like

**Comment by adelton** [comment]
Running merely
```
sudo dnf module install cri-o
```
as the commit suggests fails on Fedora 30:
```
Cannot enable more streams from module 'cri-o' at the same time
Unable to resolve argument cri-o
Error: Problems in request:
```
The module needs to be versioned ... which is sort of the problem because on each OS version, the module version will be different.

**Comment by haircommander** [comment]
> The module needs to be versioned ... which is sort of the problem because on each OS version, the module version will be different.

fixed

@TomSweeneyRedHat @rhatdan @mrunalp @umohnani8 PTAL

**Comment by adelton** [comment]
The `dnf module install cri-o:$VERSION` command fails on latest Fedora 30:

```
Last metadata expiration check: 0:58:13 ago on Thu 17 Oct 2019 02:35:08 PM IST.
No default profiles for module cri-o:1.14. Available profiles: default
Error: Problems in request:
missing groups or modules: cri-o:1.14
```

Downgrading `dnf` seems to make the same command pass.

**Comment by haircommander** [comment]
my goodness :roll_eyes: sorry for all of this @adelton . I opened an issue here https://github.com/cri-o/cri-o/issues/2897

**Comment by adelton** [comment]
It seems that due to https://bugzilla.redhat.com/show_bug.cgi?id=1751765, the steps on Fedora 30 need to be
```
VERSION=1.14
dnf module enable cri-o:$VERSION
dnf install cri-o
```

**Pull request merged**